qemu/tests/cfg/migrate.cfg

```diff
@@ -14,14 +14,22 @@
     # vmstate_check = yes
     variants:
         - @default:
+        - with_multifd:
+            only tcp,unix,fd
+            migrate_capabilities = "{'x-multifd': 'on'}"
```

Comment:
Not quit sure, but  here https://wiki.qemu.org/Features/Migration-Multiple-fds shows `x-multiple-fds`

Reply:
@vivianQizhu Thank you for being careful, I think the wiki is outdated. I got chance to test the qemu upstream patch.

Reply:
@balamuruhans Okay, thanks confirming and sharing it.